test/test_masked.py

```diff
@@ -162,6 +162,22 @@ def apply_masked_normalization_along_dim(op, input, *args, **kwargs):
 
 masked_ops = [op for op in op_db if op.name.startswith('_masked.')]
 masked_ops_with_references = [op for op in masked_ops if op.name.rsplit('.', 1)[-1] in reference_functions]
+masked_ops_with_mixed_layouts_support = [op for op in masked_ops if op.supports_sparse or op.supports_sparse_csr]
+
+
+def to_strided(obj):
+    """Convert the tensor content of object to strided tensor content.
+    """
+    if isinstance(obj, torch.Tensor):
+        if obj.layout == torch.strided:
```

Comment:
I don't think this is needed. to_dense should check this and avoid conversion and if we're not sure we should check that it does.

Reply:
Ideally, I agree but we have this:
```python
>>> torch.tensor([[1, 2]]).to_dense()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NotImplementedError: Could not run 'aten::to_dense' with arguments from the 'CPU' backend
```

Reply:
Ok then we should add a stub under the [CompositeExplicitAutograd key](https://github.com/pytorch/pytorch/tree/master/aten/src/ATen/native#dispatch) in a separate PR.

Reply:
cref: https://github.com/pytorch/pytorch/issues/59958